mattermost-plugin/webapp/webpack.config.js

```diff
@@ -203,8 +203,18 @@ config.plugins.push(new webpack.DefinePlugin({
 }));
 
 if (NPM_TARGET === 'start:product') {
+    const url = new URL(process.env.MM_BOARDS_DEV_SERVER_URL ?? 'http://localhost:9006');
+
+    for (const rule of config.module.rules) {
+        if (rule.type === 'asset/resource' && rule.generator) {
+            rule.generator.publicPath = url.toString() + 'static/';
+        }
+    }
```

Comment:
No, it's just doing what the line I reverted above (line 119) did before, but for arbitrary URLs. I also moved it down here so that it only applies to dev server builds